accelerate/nextjs-starter/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
there's 1,001 ways you can make a tsconfig. I recommend us standardizing on something like [the tsconfig.guide](https://tsconfig.guide) so that we don't have to think about it.

Reply:
@ankur-arch I think you created this one. wdyt about Jon's suggestion?

Reply:
I used the default one generated by create next app. 
@jharrell I think following a standard to create the tsconfigs would be great. Jon do you think I should update the tsconfig to be minimal and provide the link to the [guide](https://tsconfig.guide/) you suggested?

Reply:
I'm skipping updating the tsconfigs here. I think it's best to cross check all the tsconfigs in the repo and handle that in a separate PR. @jharrell

Reply:
> I think it's best to cross check all the tsconfigs in the repo and update that in a separate PR. 

sounds good to me 👍